docs/config.md

```diff
@@ -108,7 +108,7 @@ module.exports = {
 
 ### Directory for Includes
 
-Controls the directory inside which the template includes/extends/partials/etc can be found.
+Controls which directory is the _includes_ directory. The _includes_ directory is meant for templates and data in general, like layout templates.
```

Comment:
I'm not sure it _is_ meant for 'data in general'. Is it?

Reply:
No, it is not.
What I meant was _template_ data in general. I wrote it in a hurry.